es-app/src/FileSorts.cpp

```diff
@@ -40,8 +40,18 @@ namespace FileSorts
 	bool compareName(const FileData* file1, const FileData* file2)
 	{
 		// we compare the actual metadata name, as collection files have the system appended which messes up the order
-		std::string name1 = file1->metadata.get("name");
-		std::string name2 = file2->metadata.get("name");
+		std::string name1;
+		std::string name2;
+                if(file1->metadata.get("sortname").empty()){
+		  name1 = file1->metadata.get("name");
```

Comment:
Please use a tab to indent these if clauses.

Reply:
Sorry, I'm typically a space indenter, but I can fix this for constancy.  I'll try to get to that sometime tomorrow.

Reply:
Whitespace fixed